src/node_util.cc

```diff
@@ -116,19 +113,15 @@ static void GetPromiseDetails(const FunctionCallbackInfo<Value>& args) {
 
 static void GetProxyDetails(const FunctionCallbackInfo<Value>& args) {
   // Return undefined if it's not a proxy.
-  if (!args[0]->IsProxy())
-    return;
+  if (!args[0]->IsProxy()) return;
 
   Local<Proxy> proxy = args[0].As<Proxy>();
 
   // TODO(BridgeAR): Remove the length check as soon as we prohibit access to
   // the util binding layer. It's accessed in the wild and `esm` would break in
   // case the check is removed.
   if (args.Length() == 1 || args[1]->IsTrue()) {
-    Local<Value> ret[] = {
-      proxy->GetTarget(),
-      proxy->GetHandler()
-    };
+    Local<Value> ret[] = {proxy->GetTarget(), proxy->GetHandler()};
```

Comment:
There are a large number of unrelated changes in here. Were these manually edited or did the format tool do these?

Reply:
All of the formatting changes were done by saving the file (after configuring vscode with the settings example provided by the contributing guide). Not sure exactly _why_ so many changes happened, but running the cpp formatter through make also didn't revert it. I can manually revert it if desired

Reply:
So, I did revert these changes, and staged them, but after running `make format-cpp`, the file was reformatted to how it is on this PR. I'll push the non-formatted change but not sure if I should open a PR after formatting the file or just leave it be

Reply:
Yet running `CLANG_FORMAT_START=main make format-cpp` changes nothing, confusing.. I'll leave that be for now